Source/Alamofire.swift

```diff
@@ -91,279 +95,292 @@ extension URLRequest {
+    return SessionManager.default.request(urlRequest.urlRequest)
 }
 
-// MARK: - Upload Methods
-
-// MARK: File
-
-/**
-    Creates an upload request using the shared manager instance for the specified method, URL string, and file.
+// MARK: - Download Request
 
-    - parameter method:    The HTTP method.
-    - parameter URLString: The URL string.
-    - parameter headers:   The HTTP headers. `nil` by default.
-    - parameter file:      The file to upload.
+// MARK: URL Request
 
-    - returns: The created upload request.
-*/
+/// Creates a download `Request` using the default session manager instance to retrieve the contents of the specified
+/// `method`, `urlString`, `parameters`, `encoding`, `headers` and save them to the `destination`.
+///
+/// - parameter urlString:   The URL string.
+/// - parameter destination: The closure used to determine the destination of the downloaded file.
+/// - parameter method:      The HTTP method.
+/// - parameter parameters:  The parameters. `nil` by default.
+/// - parameter encoding:    The parameter encoding. `.URL` by default.
+/// - parameter headers:     The HTTP headers. `nil` by default.
+///
+/// - returns: The created download `Request`.
 @discardableResult
-public func upload(
-    _ method: Method,
-    _ URLString: URLStringConvertible,
-    headers: [String: String]? = nil,
-    file: URL)
+public func download(
+    _ urlString: URLStringConvertible,
+    to destination: Request.DownloadFileDestination,
+    withMethod method: HTTPMethod,
+    parameters: [String: AnyObject]? = nil,
+    encoding: ParameterEncoding = .url,
+    headers: [String: String]? = nil)
     -> Request
 {
-    return Manager.sharedInstance.upload(method, URLString, headers: headers, file: file)
+    return SessionManager.default.download(
+        urlString,
+        to: destination,
+        withMethod: method,
+        parameters: parameters,
+        encoding: encoding,
+        headers: headers
+    )
 }
 
-/**
-    Creates an upload request using the shared manager instance for the specified URL request and file.
+/// Creates a download `Request` using the default session manager instance to retrieve the contents of the specified
+/// `urlRequest` and save them to the `destination`.
+///
+/// - parameter urlRequest:  The URL request.
+/// - parameter destination: The closure used to determine the destination of the downloaded file.
+///
+/// - returns: The created download `Request`.
+@discardableResult
+public func download(
+    _ urlRequest: URLRequestConvertible,
+    to destination: Request.DownloadFileDestination)
+    -> Request
+{
+    return SessionManager.default.download(urlRequest, to: destination)
+}
 
-    - parameter URLRequest: The URL request.
-    - parameter file:       The file to upload.
+// MARK: Resume Data
 
-    - returns: The created upload request.
-*/
+/// Creates a download `Request` using the default session manager instance from the `resumeData` produced from a
```

Comment:
Do we always need to say "default session manager instance"? Would "default session manager" or "default `SessionManager`" be enough? There are multiple instances of this phrase.